web/src/scheduled_messages.js

```diff
@@ -319,3 +323,40 @@ export function update_send_later_options() {
         $("#send_later_options").replaceWith(render_send_later_modal_options(filtered_send_opts));
     }
 }
+
+export function show_schedule_message_indicator() {
+    const scheduled_messages = Object.values(scheduled_messages_data);
+
+    const matching_scheduled_messages = scheduled_messages.filter((scheduled_message) => {
+        if (scheduled_message.failed === true) {
+            return false;
+        }
+        switch (scheduled_message.type) {
+            case "private":
+                if (scheduled_message.to.toString() === narrow_state.pm_ids_string()) {
+                    return true;
+                }
+                break;
+            case "stream":
+                if (
+                    scheduled_message.topic === narrow_state.topic() &&
+                    scheduled_message.to === narrow_state.stream_sub().stream_id
+                ) {
+                    return true;
```

Comment:
This logic is wrong -- topics as case-insensitive. You want to use `util.same_stream_and_topic`